Add missing zdbId

TobiasNx opened the pull request.

blackwinter reviewed (approved):
I suppose it's okay, so :+1:

(For future reference, `035$$a` with prefix `(DE-599)` is the [EKI](https://verbundwiki.gbv.de/display/VZG/Erstkatalogisierungs-ID).)

But while you're at it, you might also want to map `016$7#$a` with `$$2=DE-600`.

TobiasNx commented:
Added the requested transformation which indeed catched a missing zdbId. I also added an dnbId and an object for dnb resources since they are provided in 016 too.

TobiasNx commented:
@acka47 is it okay to map the dnbId of the record too?

dr0i commented on src/test/resources/alma-fix/990103899140206441.json:
```diff
@@ -3,6 +3,7 @@
   "almaMmsId" : "990103899140206441",
   "oclcNumber" : [ "1367753905" ],
   "zdbId" : "590016-5",
+  "dnbId" : "01335941X",
```
This field `dnbId` has no entry in the `context.jsonld`.
Can you provide one ?

TobiasNx replied:
I added it now

TobiasNx commented:
@acka47 approved this

dr0i commented:
Will be deployed next Monday.
